contracts/wallet.sol

```diff
@@ -250,7 +250,7 @@ library DailyLimitTrait {
         uint available;
         uint limitTimestamp;
         uint pending;
-        bool updateable;
+        bool ownerUpdateable;
```

Comment:
I may be wrong, but I think `ownerUpdatable` is the opposite of what this means.  When this is `false` the owner can still set the limit when this is `true` then it can only be updated by the 2FA process.

I think that a name like `2faRequired` or `controllerConfirmationRequired` is a better choice.

Reply:
yes, you are right, `updateable` refers to 2FA, so the logic should change to make it `ownerUpdateable`

Reply:
I guess the question here is: 
What is the semantics of `updateable` : curently it tells you whether not an update requires a controller to confirm it. If so, it is a terrible name, I am to blame here. 

I think the right thing is to updateable to equal "ownerUpdateable" and as a result the public functions that use it should change name. 

But if we want to change the variable to something like controllerConfirmationRequired then i would rather change the names of the public functions too. 

I have a preference but if the cost of creating new APIs is high on our backend ... @i-stam and @jeffreybolle

Reply:
I think that what you have written makes perfect sense, because updating the internal variable name and not the external function's name will still cause the confusion it caused before.

Reply:
rethinking this, I actually agree.  We should change both and just handle that change in the backend.

For version 4 of the contract we should sit down and design the interface of our contract as separate from the contract itself and then commit to not changing that interface again in the future (except to add to it).

Reply:
IMO we should use semantic versioning and assume that new major version of the contract breaks the ABI.

Reply:
yeah, that was the plan with the version numbers .... and we have just gone from a v2.X.X to a v3.X.X ... but yeah, we need to be better at this, it is hard with immutable software.

Reply:
as discussed, we are going to keep the semantics of the variable whilst renaming accordingly. We are going for `controllerConfirmationRequired`. And we have updated the names of the getter methods accordingly too.